README.md

```diff
@@ -106,9 +106,11 @@ DEFAULT_INSTANCE_BASE_DOMAIN='example.com'
 GANDI_API_KEY='api-key'
 GITHUB_ACCESS_TOKEN='github-token'
 WATCH_ORGANIZATION='github-org'
-RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
-RABBITMQ_ADMIN_USERNAME='admin'
-RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
+DEFAULT_RABBITMQ_ADMIN_USERNAME='admin'
+DEFAULT_RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_INSTANCE_RABBITMQ_HOST='rabbitmq-dev.opencraft.hosting'
```

Comment:
Also don't think we should be setting a default value at all for the base URL - it's ok to make everyone at OC override it to use `rabbitmq-dev.opencraft.hosting` -- that's what our dev `.env` file is for.